src/core/math/line3/fromData.js

```diff
@@ -0,0 +1,13 @@
+
+/**
+ * Create a line in 3D space from the given data.
+ *
+ * and point is a random point on the line
+ * where direction must be a unit vector
+ */
+const fromData = (point, direction) => {
```

Comment:
I think we should rename to `fromPointAndDirection` (it is waay more clear what it does)

Reply:
Done